test/parallel/test-https-agent.js

```diff
@@ -37,7 +38,7 @@ server.listen(0, function() {
         }, function(res) {
           res.resume();
           console.log(res.statusCode);
-          if (++responses == N * M) server.close();
+          if (++responses === N * M) server.close();
         }).on('error', function(e) {
           console.log(e.message);
```

Comment:
I wonder if it's better to throw the error here instead of logging the error message. Maybe the whole `error` listener can be removed. An error is thrown if there isn't one.